pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/dto/PatientDTO.java

```diff
@@ -24,13 +24,15 @@
 	
 	public PatientDTO() {
 		orgMaps = new ArrayList<PatientOrganizationMapDTO>();
+		patientName = new PatientNameDTO();
```

Comment:
A Patient can have more than one PatientName. For here (and in the PatientRecord, and anywhere else the PatientName is referenced) this should probably be something like `patientName = new ArrayList<PatientNameDTO>();`